src/handlers/input.rs

```diff
@@ -9,11 +9,37 @@ use unicode_width::{UnicodeWidthChar, UnicodeWidthStr};
 // Handle event when the search input block is active
 pub fn handler(key: Key, app: &mut App) {
   match key {
+    Key::Ctrl('k') => {
+      app.input.drain(app.input_idx..app.input.len());
+    }
     Key::Ctrl('u') => {
+      app.input.drain(..app.input_idx);
+      app.input_idx = 0;
+      app.input_cursor_position = 0;
+    }
+    Key::Ctrl('c') => {
```

Comment:
`Ctrl('c')` is the global exit key of the program (used [here](https://github.com/Rigellute/spotify-tui/blob/master/src/util.rs#L24)).

What other keybind would be appropriate here instead?